build.sh

```diff
@@ -81,6 +81,14 @@ for BINARY in $BINARIES; do
     error "Unable to find output"
     exit 1
   fi
+  
+  # MINIFY set to false by default
+  MINIFY=${MINIFY:-"false"}
+  if [ "$MINIFY" = "true" ]; then
```

Comment:
Some kind of check needs to be added for whether strip and upx work for the target triple

Reply:
Is it solved by e3b5b29 ?

Reply:
Because we use `set -e` the script immediately exits when any command returns a non-zero error code, these if statements will never run. Using `|| info Unable to strip binary` would be a solution that would work.